### PR TITLE
New tables and add foreign key 

### PR DESCRIPTION
**Pull request by Hassaanjbaig-code**
## In this pull request 

- Create a table of owner and species 
- Modify the animal according to the [requirement ](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md)
- Insert the owner's data and species 
-  Modify and insert data species_id and owner_id.
- Use Join

image

image

image

image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -39,4 +39,18 @@ select count(animal_escape_attempts) from animal group by neutered;
...
+
+select animal.owner_id, animal.animalname, owners.fullname, owners.id from animal join owners on animal.owner_id = owners.id where owners.fullname = 'Melody Pond';
+
+select animal.species_id, animal.animalname, species.id, species.fullname from animal join species on animal.species_id = species.id where species.fullname = 'Pokemon';
+
+select animal.owner_id, animal.animalname, owners.fullname, owners.id from animal join owners on animal.owner_id = owners.id where animal.owner_id = null;
+
+select count(animal.species_id), species.id, species.fullname from animal join species on animal.species_id = species.id group by species.id;
+
+select species.id, owners.id, owners.fullname from animal join species on animal.species_id  =species.id join owners on animal.owner_id = owners.id where owners.fullname = 'Jennifer Orwell';
```
Comment (AnselemOdims) [inline queries.sql:57]: - [ ] Good job working on the queries but this does not quite fully represent the statement `List all Digimon owned by Jennifer Orwell.`

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Good job so far but please kindly add screenshots for all queries to your pull request description
- [x] Please kindly include a query for the question `Who owns the most animals?`
- [x] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 1 above)
Comment (Hassaanjbaig-code) [reply queries.sql:57]: @AnselemOdims  Can you explain what should I use in here 

Thank you

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @Hassaanjbaig-code,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Hassaanjbaig-code** [comment]
@Kossi-stack Thank you for review

**Pull request merged**